doc/tests.rst

```diff
@@ -288,6 +288,21 @@ a ``deps`` field the ``inline_tests`` field. The argument of this
     (inline_tests (deps data.txt))
     (preprocess (pps ppx_expect)))
 
+Specifying Inline Test arguments for Parameterised Libraries
+------------------------------------------------------------
+
+If your library is parameterised (see
+:doc:`/reference/dune/library_parameter`), you must specify which
+implementation of the parameters to use with the ``arguments`` field:
+
+.. code:: ocaml
+
+   (library
+    (name foo)
+    (parameters a_param b_param)
+    (inline_tests
+      (arguments a_impl b_impl)))
+
```

Comment:
Just a suggstion, feel free to disregard:

I think it can help with readability to accompany examples with descriptions how how to map the example to the general concept:

```suggestion
implementation of the parameters to use with the ``arguments`` field. E.g.,
if `foo` is a parameterised library, taking parameters `a_param` and
`b_param`, you can specify the implementations to use for the parameters for
inline tests as follows:

.. code:: ocaml

   (library
    (name foo)
    (parameters a_param b_param)
    (inline_tests
      (arguments a_impl b_impl)))

```

IMO, this can leave the reader with less to puzzle out, tho it does come at the cost of redundancy.

Reply:
Thanks I like that better indeed!